src/Muscles/Muscle.cpp

```diff
@@ -267,7 +267,7 @@ void muscles::Muscle::setState(
     if (emg.type() == muscles::STATE_TYPE::BUCHANAN) {
         m_state = std::make_shared<muscles::StateDynamicsBuchanan>
                   (muscles::StateDynamicsBuchanan());
-    } else if (emg.type() == muscles::STATE_TYPE::DE_GROOTE) {
+    } else if (emg.type() == muscles::STATE_TYPE::DE_GROOTE_STATE) {
```

Comment:
Why specifying `_STATE`?

Reply:
Cause I added a muscle type named "de_groote" so one name had to be changed.

Reply:
Why? One is a "STATE_TYPE" and the other is a "MUSCLE_TYPE"

Reply:
Is this because of Python binder?

Reply:
I don't remember but i had some compilation issues.